_layouts/repo_instance.html

```diff
@@ -75,34 +75,34 @@
         <div class="panel panel-default">
           <div class="panel-heading"><h3 class="panel-title">CONTRIBUTING</h3></div>
           <div class="panel-body">
-            {% if page.repo.snapshots[distro].data.contributing %}
```

Comment:
You have eliminated the data.contributing attribute, but this is still being generated in rosindex_generator.rb.  So you need to remove there:
```      'contributing' => nil,```
as well as change
```data['contributing_rendered'], data['contributing'] = get_contributing(  ... ```
to
```data['contributing_rendered'] = get_contributing( ...`